src/Microsoft.Health.Fhir.Api/Controllers/FhirController.cs

```diff
@@ -424,14 +424,14 @@ public async Task<IActionResult> SearchCompartmentByResourceType(string compartm
 
         private async Task<IActionResult> PerformCompartmentSearch(string compartmentType, string compartmentId, string resourceType, IReadOnlyList<Tuple<string, string>> queries)
         {
-            Bundle response = await _mediator.SearchResourceCompartmentAsync(compartmentType, compartmentId, resourceType, queries, HttpContext.RequestAborted);
+            var response = await _mediator.SearchResourceCompartmentAsync(compartmentType, compartmentId, resourceType, queries, HttpContext.RequestAborted);
```

Comment:
This should remain `Bundle` according to our coding guideline since the right-hand side does not clearly identify the object type.